python/examples/demo.py

```diff
@@ -29,23 +29,23 @@
 
 spa.file("in.demo")
```

Comment:
The "b" before the string is needed here due to the different string treatment in Python3; otherwise it does not work with Python3.

Reply:
for more information, see e.g.:
https://stackoverflow.com/questions/2592764/what-does-a-b-prefix-before-a-python-string-mean

In Python3 the string syntax has changed, but with the included __future__ statements it is backward compatible to Python2 > 2.6.

Reply:
>The "b" before the string is needed here due to the different string treatment in Python3; otherwise it does not work with Python3.

I see, sorry about that. I'm going to defer to @sjplimp on this pull request.